plugins/content/caption/base/lib/model/CaptionAsset.php

```diff
@@ -79,11 +79,48 @@ public function getName()
 	{
 		return $this->getLanguage();
 	}
-	
+
+	protected function validateCaptionContent()
+	{
+		KalturaLog::debug("Validate caption content for caption asset ID {$this->getId()}");
+
+		//Get the Content Manager
+		$captionsContentManager = kCaptionsContentManager::getCoreContentManager($this->getContainerFormat());
+		if (!$captionsContentManager)
+		{
+			KalturaLog::debug("Failed to find content manager for caption ID {$this->getId()} and container format {$this->getContainerFormat()}");
```

Comment:
Same here - its not a debug message but a WARNING (since it may not have plugin enabled)